### ISSUE TITLE
vue/valid-v-slot ignores "off" setting in .eslintrc.js

### ISSUE THREAD

**Issue by ffxsam** [vuejs/eslint-plugin-vue#1269]
**Tell us about your environment**
* **ESLint version:** 6.8.0
* **eslint-plugin-vue version:** 6.2.2
* **Node version:** 12.13.1

**Please show your full configuration:**

```js
/* eslint-disable max-len */
module.exports = {
  root: true,
  env: {
    node: true,
  },
  extends: [
    'plugin:vue/recommended',
    '@vue/standard',
    '@vue/typescript/recommended',
  ],
  parserOptions: {
    ecmaVersion: 2020,
  },
  rules: {
    'array-callback-return': 'error',
    'arrow-spacing': ['error', { before: true, after: true }],
    'comma-dangle': ['error', 'always-multiline'],
    curly: ['error', 'multi-line'],
    'default-case': 'error',
    'dot-location': ['error', 'property'],
    'dot-notation': 'error',
    eqeqeq: 'warn',
    'key-spacing': 'error',
    indent: ['error', 2, { SwitchCase: 1 }],
    'max-len': ['warn', { code: 80 }],
    'newline-per-chained-call': 'error',
    'no-alert': 'warn',
    'no-await-in-loop': 'error',
    'no-case-declarations': 'error',
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'warn',
    'no-const-assign': 'error',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'warn',
    'no-duplicate-imports': 'error',
    'no-else-return': 'error',
    'no-implicit-coercion': ['error', { allow: ['!!'] }],
    'no-mixed-spaces-and-tabs': 'error',
    'no-multi-spaces': 'error',
    'no-param-reassign': 'error',
    'no-return-await': 'error',
    'no-sequences': 'error',
    'no-trailing-spaces': 'error',
    'no-undef-init': 'error',
    'no-unreachable': 'warn',
    // 'no-unused-vars': 'warn',
    'no-useless-return': 'warn',
    'no-var': 'error',
    'object-curly-spacing': [
      'error',
      'always',
      { arraysInObjects: true, objectsInObjects: true },
    ],
    'object-shorthand': ['error', 'always'],
    'prefer-const': 'error',
    'prefer-promise-reject-errors': 'warn',
    quotes: [
      'error',
      'single',
      {
        avoidEscape: true,
      },
    ],
    semi: ['error', 'always'],
    'space-before-function-paren': [
      'error',
      {
        anonymous: 'never',
        named: 'never',
        asyncArrow: 'always',
      },
    ],
    'spaced-comment': ['error', 'always', {}],
    yoda: 'error',

    // Vue rules
    'vue/html-closing-bracket-spacing': [
      'warn',
      {
        selfClosingTag: 'never',
      },
    ],
    'vue/html-closing-bracket-newline': [
      'error',
      {
        singleline: 'never',
        multiline: 'always',
      },
    ],
    'vue/no-unused-components': 'warn',
    'vue/no-unused-vars': 'off',
    'vue/valid-v-slot': 'off',

    // TypeScript rules
    // '@typescript-eslint/array-type': ['error', 'array'],
    // '@typescript-eslint/ban-types': 'error',
    // '@typescript-eslint/camelcase': 'error',
    // '@typescript-eslint/class-name-casing': 'error',
    '@typescript-eslint/explicit-function-return-type': 'off', // TODO: remove later
    '@typescript-eslint/explicit-member-accessibility': 'off', // TODO: remove later
    '@typescript-eslint/indent': ['error', 2, { SwitchCase: 1 }],
    '@typescript-eslint/no-explicit-any': 'off', // TODO: remove later
    '@typescript-eslint/ban-ts-ignore': 'off',
    // '@typescript-eslint/no-unused-vars': 'warn',
    // '@typescript-eslint/type-annotation-spacing': 'error',
  },
  overrides: [
    {
      files: [
        '**/__tests__/*.{j,t}s?(x)',
        '**/tests/unit/**/*.spec.{j,t}s?(x)',
      ],
      env: {
        jest: true,
      },
    },
  ],
};
```

**What did you do?**	

```vue
<template #item.play="{ item }">
  ...
</template>
```

**What did you expect to happen?**
Not show a linting error for the code above.

**What actually happened?**

```json
{
	"resource": "/Volumes/SuperData/Sites/reelcrafter/v2-frontend/src/views/Tracks.vue",
	"owner": "_generated_diagnostic_collection_name_#1",
	"severity": 8,
	"message": "[vue/valid-v-slot]\n'v-slot' directive doesn't support any modifier.",
	"source": "eslint-plugin-vue",
	"startLineNumber": 23,
	"startColumn": 23,
	"endLineNumber": 23,
	"endColumn": 44
}
```

**Comment by ota-meshi** [comment]
That warning is displayed by eslint-plugin-vue built into Vetur.
Vetur's built-in eslint-plugin-vue does not reference configuration file.

You need to add `"vetur.validation.template": false` to your `.vscode/settings.json`.

https://vuejs.github.io/vetur/linting-error.html#linting
https://eslint.vuejs.org/user-guide/#visual-studio-code
